From 9168a9ba147af738a3a8cf05708a541b2d5f4bc4 Mon Sep 17 00:00:00 2001
From: Don <don.j.olmstead@gmail.com>
Date: Thu, 3 Jan 2019 16:54:07 -0800
Subject: [PATCH 2/3] Do not append static to library name

On Windows a static lib will have _static appended. In vcpkg only a shared or static library is created so remove this behavior.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a11ddf0f4..cbb9f541b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -632,7 +632,7 @@ else()
   # We also need to use a custom suffix, in order to distinguish between the
   # shared import library name and the static library name.
   set(PNG_SHARED_OUTPUT_NAME "libpng${PNGLIB_ABI_VERSION}")
-  set(PNG_STATIC_OUTPUT_NAME "libpng${PNGLIB_ABI_VERSION}_static")
+  set(PNG_STATIC_OUTPUT_NAME "libpng${PNGLIB_ABI_VERSION}")
 endif()
 
 if(PNG_SHARED)
-- 
2.41.0.windows.1

